app/src/types/user.js

```diff
@@ -42,13 +44,17 @@ export type User = t.TypeOf<typeof UserModel>;
  * @param from
  * @returns {User}
  */
-export function extractUserFromRequest(from: express$Request): User {
+export function extractUserFromRequest(from: express$Request): Option<User> {
   const reqWithUser = ((from: Object): { user: User });
 
-  // eslint-disable-next-line no-unused-vars
-  return t.validate(reqWithUser.user, UserModel).fold(_ => {
-    return createEmptyUser();
-  }, t.identity);
+  return t.validate(reqWithUser.user, UserModel).fold(
```

Comment:
`validate` returns an [Either](https://github.com/gcanti/fp-ts/blob/master/src/Either.ts) that provides a `toOption` method, you can replace the `fold` with `t.validate(reqWithUser.user, UserModel).toOption()`

Reply:
you may actually want to return the `Either` to the caller (instead of the `Option`) as the `Either` will provide validation errors that you can return to the API client, as we do [here](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/controllers/messages.ts#L116)